client/app/views/main.coffee

```diff
@@ -128,6 +128,36 @@ module.exports = class HomeView extends BaseView
         @displayView @configApplications
         window.document.title = t "cozy applications title"
 
+    displayUpdateApplication: (slug) =>
+        @displayView @configApplications
+        window.document.title = t "cozy applications title"
+        window.app.routers.main.navigate 'config-applications', false
+
+        # When the route is called on browser loading, it must wait for
+        # apps list to be retrieved
+        if @apps.length is 0
```

Comment:
(I have just checked) The user is redirected to "#config-applications" page, no error. Also, this shouldn't happen since this route is only accessible through update notifications.